src/adservice/settings.gradle

```diff
@@ -1 +1 @@
-rootProject.name = 'hipstershop'
+rootProject.name = 'opentelemetry-demo-ad-service'
```

Comment:
Giving the adservice module a more appropriate name. This property is used by IDEs as the name of the project, so this makes it easier to find in a list. `Dockerfile` needed a small update as well.